examples/extformer_moe/conf/extformer_moe_enso_pretrain.yaml

```
@@ -0,0 +1,154 @@
+hydra:
```

我们在ppsci.utils.config里为hydra设置一些默认参数，可以作如下修改

```suggestion
defaults:
  - ppsci_default
  - TRAIN: train_default
  - TRAIN/ema: ema_default
  - TRAIN/swa: swa_default
  - EVAL: eval_default
  - INFER: infer_default
  - hydra/job/config/override_dirname/exclude_keys: exclude_keys_default
  - _self_

hydra:
```